src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -555,8 +565,51 @@
 	text-align: center;
 	text-decoration: none;
 	width: 233px;
+
+	&:hover {
+		background-color: var(--tec-color-button-primary-hover);
+		color: #fff;
+	}
+
+	&:active {
+		background-color: var(--tec-color-button-primary-active);
+		color: #fff;
+	}
```

Comment:
In order to follow what we do in other places, I'd group the color in `&:hover, &:active` and then use the individuals for the `background-color`.